openjdk/README.md

```diff
@@ -0,0 +1,33 @@
+openjdk
+==========
+This provides a Clear Linux* openjdk container
+
+Build
+-----
+```
+docker build -t clearlinux/openjdk .
+```
+
+Or just pull it from Dockerhub
+---------------------------
+```
+docker pull clearlinux/openjdk
+```
+
+Compile your java app inside the Docker container
+-----------------------
+```
+docker run --rm -it --name myapp -v "$PWD":/usr/src/myapp -w /usr/src/myapp clearlinux/openjdk javac myapp.java
+```
+
+Run your java app instance
+-----------------------
+```
+docker run --rm -it --name myapp -v "$PWD":/usr/src/myapp -w /usr/src/myapp clearlinux/openjdk java myapp
+```
+
```

Comment:
Please add a link to docker hub official openjdk as mentioned in the CONTRIBUTING.md.